lib/oli_web/controllers/resource_controller.ex

```diff
@@ -19,37 +19,65 @@ defmodule OliWeb.ResourceController do
   def edit(conn, %{"project_id" => project_slug, "revision_slug" => revision_slug}) do
     author = conn.assigns[:current_author]
     is_admin? = Accounts.is_admin?(author)
-    project = conn.assigns.project
 
     case PageEditor.create_context(project_slug, revision_slug, conn.assigns[:current_author]) do
       {:ok, context} ->
-        render(conn, determine_editor(context),
-          active: :curriculum,
-          breadcrumbs:
-            Breadcrumb.trail_to(project_slug, revision_slug, Oli.Publishing.AuthoringResolver),
-          is_admin?: is_admin?,
-          context: Jason.encode!(context),
-          raw_context: context,
-          scripts: Activities.get_activity_scripts(:authoring_script),
-          part_scripts: PartComponents.get_part_component_scripts(:authoring_script),
-          project_slug: project_slug,
-          revision_slug: revision_slug,
-          activity_types: Activities.activities_for_project(project),
-          part_component_types: PartComponents.part_components_for_project(project),
-          graded: context.graded
-        )
+        render_editor(context, conn, project_slug, revision_slug, is_admin?)
```

Comment:
Before, we were rendering the editor for both types of page with the same assigns, which seemed weird since the 2 templates required different properties and it was hard to tell which property was for which template.